resources/resources_autogenerated.qrc

```diff
@@ -1,122 +1,122 @@
-        <file>contributors.txt</file>
-        <file>emoji.json</file>
-        <file>error.png</file>
-        <file>examples/moving.gif</file>
-        <file>examples/splitting.gif</file>
-        <file>icon.ico</file>
-        <file>icon.png</file>
-        <file>licenses/boost_boost.txt</file>
-        <file>licenses/emoji-data-source.txt</file>
-        <file>licenses/libcommuni_BSD3.txt</file>
-        <file>licenses/lrucache.txt</file>
-        <file>licenses/magic_enum.txt</file>
-        <file>licenses/openssl.txt</file>
-        <file>licenses/pajlada_settings.txt</file>
-        <file>licenses/pajlada_signals.txt</file>
-        <file>licenses/qt_lgpl-3.0.txt</file>
-        <file>licenses/qtkeychain.txt</file>
-        <file>licenses/rapidjson.txt</file>
-        <file>licenses/websocketpp.txt</file>
-        <file>pajaDank.png</file>
-        <file>qss/settings.qss</file>
-        <file>scrolling/downScroll.png</file>
-        <file>scrolling/downScroll.svg</file>
-        <file>scrolling/neutralScroll.png</file>
-        <file>scrolling/neutralScroll.svg</file>
-        <file>scrolling/upScroll.png</file>
-        <file>scrolling/upScroll.svg</file>
-        <file>settings/about.svg</file>
-        <file>settings/aboutlogo.png</file>
-        <file>settings/accounts.svg</file>
-        <file>settings/advanced.svg</file>
-        <file>settings/behave.svg</file>
-        <file>settings/browser.svg</file>
-        <file>settings/commands.svg</file>
-        <file>settings/emote.svg</file>
-        <file>settings/externaltools.svg</file>
-        <file>settings/filters.svg</file>
-        <file>settings/ignore.svg</file>
-        <file>settings/keybinds.svg</file>
-        <file>settings/moderation.svg</file>
-        <file>settings/notification2.svg</file>
-        <file>settings/notifications.svg</file>
-        <file>settings/theme.svg</file>
-        <file>sounds/ping2.wav</file>
-        <file>split/down.png</file>
-        <file>split/left.png</file>
-        <file>split/move.png</file>
-        <file>split/right.png</file>
-        <file>split/up.png</file>
-        <file>streamerMode.png</file>
-        <file>switcher/plus.svg</file>
-        <file>switcher/popup.svg</file>
-        <file>switcher/switch.svg</file>
-        <file>tlds.txt</file>
-        <file>twitch/admin.png</file>
-        <file>twitch/automod.png</file>
-        <file>twitch/broadcaster.png</file>
-        <file>twitch/cheer1.png</file>
-        <file>twitch/globalmod.png</file>
-        <file>twitch/moderator.png</file>
-        <file>twitch/prime.png</file>
-        <file>twitch/staff.png</file>
-        <file>twitch/subscriber.png</file>
-        <file>twitch/turbo.png</file>
-        <file>twitch/verified.png</file>
-        <file>twitch/vip.png</file>
-        <file>buttons/pinDisabledDark.png</file>
-        <file>buttons/pinDisabledLight.png</file>
-        <file>buttons/pinEnabled.png</file>
-    </qresource>
```

Comment:
This seems to be what happens when adding the resource through qtcreator. Didn't realize there was a script to do this.